ports/xnnpack/portfile.cmake

```
@@ -0,0 +1,33 @@
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+endif()
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO google/XNNPACK
+    REF 2e42787663028a83c30dfaaa1c6f1d1a9f08d80a
+    SHA512 368046fb8d9386fa60bca97387857cfd7cda4cd07bfbd7f394e927df0b35fc1027981152da8827f282505279d50c6e421a430b59f6357621bc4e140452eca51a
+    HEAD_REF master
+    PATCHES
+        use-packages.patch
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH ${SOURCE_PATH}
```
We prefer to use `vcpkg_cmake_configure` for new ports instead of the deprecated one. Could you please update this?